go.mod

```diff
@@ -6,7 +6,9 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/containers/image/v5 v5.11.0
 	github.com/openshift/api v0.0.0-20200326160804-ecb9283fe820
-	github.com/openshift/build-machinery-go v0.0.0-20200211121458-5e3d6e570160
+	github.com/openshift/build-machinery-go v0.0.0-20210712174854-1bb7fd1518d3
 	github.com/stretchr/testify v1.7.0
-	k8s.io/apimachinery v0.20.1
+	k8s.io/apimachinery v0.22.1
```

Comment:
`go.mod` changes should be captured in a separate commit, with the title like "bump(x):"

In this case, I would use the commit title "bump(*):", with a commit message along the lines of "Updating apimachinery to v0.22.1 and build-machinery-go to latest"